platform/android/doc_classes/EditorExportPlatformAndroid.xml

```diff
@@ -615,6 +615,10 @@
 		<member name="version/name" type="String" setter="" getter="">
 			Application version visible to the user. Falls back to [member ProjectSettings.application/config/version] if left empty.
 		</member>
+		<member name="wear_os/swipe_to_dismiss" type="bool" setter="" getter="">
+			If [code]true[/code], [url=https://developer.android.com/design/ui/wear/guides/components/swipe-to-dismiss]Swipe to dismiss[/url] will be enabled on Wear OS.
+			[b]Note:[/b] This is [code]true[/code] by default.
```

Comment:
Let's add a note that this also requires `gradle build`.

Reply:
I have added a warning stating that this can only be disabled if Gradle build is enabled. Is it still necessary to include a note here?

Reply:
I added the note about the gradle_build requirement.